build.gradle

```diff
@@ -103,6 +107,30 @@ javadoc {
     }
 }
 
+
+
+if (project.hasProperty('githubToken')) {
+    System.setProperty('org.ajoberstar.grgit.auth.username', project.githubToken)
+}
+
+gitPublish {
```

Comment:
Can this be conditional on the build mode? We'd like to publish only on release builds.